docs/monthly/2023-08-monthly.md

```diff
@@ -21,54 +21,48 @@ commit and decommit in priority:
 
 #### Release 0.12.0
 
-- Support cardano-node 8.1.2
-  - Updated client and plutus versions
+- Support cardano-node v.8.1.2
```

Comment:
I believe that the cardano-node also just tags their release as `8.1.2`. So changing this back to just `8.1.2` as well.